### ISSUE TITLE
ueberzugpp stuck vifm 

### ISSUE THREAD

**Issue by dongdongbh** [vifm/vifm#879]
I'm following [vifmimg](https://github.com/thimc/vifmimg) setup ueberzug for vifm. 

When I open a pdf file, their is a thumbnail picture created in cache directory, which proof the other part works fine, but it the image do not shows up in vifm, and the vifm stucked. When I open a image, their also no image shows up, and the vifm also stucked.

I'm using [ueberzugpp](https://github.com/jstkdng/ueberzugpp), and I also posted an [issue](https://github.com/jstkdng/ueberzugpp/issues/9) their, it seems ueberzugpp works fine, their is some problem inside vifm.

### Test environment
system: `ubuntu server 20.04`
terminal: `alacritty`
vifm: 
```
Version: 0.12.1
Git info: built out of repository
Compiled at: Feb 20 2023 13:53:33

Support of extended keys is on
Parsing of .desktop files is enabled
With GTK+ library
Without magic library
With X11 library
With dynamic loading of X11 library
With file program
With -n option for cp and mv
With remote command execution
```

**Comment by xaizek** [comment]
> their is some problem inside vifm.

You're trying to use `vifmimg` with an application it's not meant to be used with. The problem is highly likely in `euberzugpp`. 

In particular, its README says:

> Layer accepts the silent [-s] option, to hide the textual output of the command.

That's the only option it accepts, while `vifmrun` also passes `-p json`, so it probably fails on the start and pipe has no reader.

**Comment by RayZ0rr** [comment]
@dongdongbh 

Earlier this week I had issues with image-preview on `ueberzug` with `vifm`. Vifm would freeze when it was showing a preview.

Right now, I am using `ueberzug` without issues on Vifm. Here's what I did to fi it.

I had an `/usr/bin/ueberzug` which was not working properly. From my memory I installed it from pip. But it didn't show association to any package managers.

It didn't show up when I did `pip list` or `sudo pacman -Qq`

So I deleted that file :P : 

`sudo trash /usr/bin/ueberzug`

**I used trash-cli instead of rm just in case**

Then I reinstalled ueberzug from official arch repos:

`sudo pacman -S ueberzug`

Then everything was back to normal
